ui/raidboss/data/05-shb/alliance/the_tower_at_paradigms_breach.js

```diff
@@ -232,6 +239,7 @@ export default {
         text: {
           en: 'Knockback -> Out of Middle',
           de: 'Rückstoß -> Raus aus der Mitte',
+          fr: 'Poussée -> Sortez du milieu',
```

Comment:
Poussée -> Éloignez-vous du milieu

Reply:
Pour le coup, j'essai de coller au plus près de la trad qui est `Hors du milieu` si je me trompe pas. Partout où il y a `Out Of...`, j'ai mis `Sortez`. Je mmet éloignez-vous quand c'est `away.....`. Si tu y tiens je peux changé, j'esaai juste de rester similaire partout.

Reply:
Le français c'est pas l'anglais, là, la map est carrée et il faut aller sur les côtés et s'éloigner du milieu. Généralement, on sort de quelque chose, et le milieu ben, c'est pas quelque chose 😊

Reply:
Si on met `Partez`, ça peut le faire ?

Reply:
Loin du milieu ?

Reply:
Je mettrais `éloignez-vous` vu qu'on voulait gardé cet aspect impératif.